ads/adverticum.md

```diff
@@ -0,0 +1,37 @@
+
+<!---
+Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+#Adverticum
+
+
+
+##Example
+
+
+<amp-ad width=300 height=250
+    type="adverticum" data-goa3zone="123456789" data-costumetargetstring="bXVzdGJlYmVhc2U2NGVuY29kZWQ=" >
+</amp-ad>
+
+Note: The only supported costume parameters are: 
```

Comment:
I would recommend add a configuration part, and list all supported parameters there.